great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -48,10 +49,12 @@ class CloudDataContext(AbstractDataContext):
 
     def __init__(
         self,
-        project_config: Union[DataContextConfig, Mapping],
-        context_root_dir: str,
-        ge_cloud_config: GeCloudConfig,
+        project_config: Optional[Union[DataContextConfig, Mapping]] = None,
+        context_root_dir: Optional[str] = None,
         runtime_environment: Optional[dict] = None,
+        ge_cloud_base_url: Optional[str] = None,
```

Comment:
Same question - could we pass an optional GeCloudConfig instead of these 3 params? or is there a reason to separate?

Reply:
Same note as above - consistency with `DataContext` constructor